articles/introduction-to-scikit-learn-in-python/index.md

```diff
@@ -0,0 +1,297 @@
+### Introduction to Scikit-Learn in Python - Part 1
+
+The concept of Machine Learning has been blooming over the past few years, and more often than not, graduate students and industry professionals have made a career switch to Data Science or Machine Learning. An essential ingredient for establishing familiarity in this field is to know your libraries and dependencies. Your work's significant chunk goes towards having the right approach towards the problem and manipulating the dataset regarding your approach. This multi-part article introduces the reader to SciKit-Learn, a vital library used to build statistical models to make predictions.
```

Comment:
Avoid unnecessary capitalization. machine learning.

Reply:
Fixed it in the recent commit.